mtp_bank_admin/apps/mtp_auth/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import url
+from django.core.urlresolvers import reverse_lazy
+
+from moj_auth import views
+
+urlpatterns = [
+    url(r'^login/$', views.login, {
+            'template_name': 'mtp_auth/login.html',
+        }, name='login'),
+    url(
+        r'^logout/$', views.logout, {
+            'template_name': 'mtp_auth/login.html',
+            'next_page': reverse_lazy('auth:login'),
+        }, name='logout'
+    ),
+]
```

Comment:
The views are in the auth app. My rationale for leaving the urls/templates out was that in order to add the templates in it would involve piling in a whole load of static files that would 9/10 be overwritten. And leaving the urls out is just because that is where you would define the templates.